stable_baselines3/common/on_policy_algorithm.py

```diff
@@ -163,7 +163,9 @@ def collect_rollouts(
             with th.no_grad():
                 # Convert to pytorch tensor or to TensorDict
                 obs_tensor = obs_as_tensor(self._last_obs, self.device)
+                self.policy.disable_training()
```

Comment:
why not just `self.policy.train()` as it is a `th.nn.Module` ?

Reply:
thanks for your immediate feedback! I'll work on it.

Reply:
Done. See commit f8ffec69f56cbe7884c511d7c610e6a8b68f8fbb.